plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/FirUtils.kt

```diff
@@ -27,3 +28,9 @@ val FirFunctionCall.functionCallArguments: List<FirExpression>
     get() = listOfNotNull(dispatchReceiver) + argumentList.arguments
 val FirFunctionSymbol<*>.effects: List<FirEffectDeclaration>
     get() = this.resolvedContractDescription?.effects ?: emptyList()
+
+/**
+ * A `FirPropertySymbol` is an **extension property** when its `classId` is `null`.
```

Comment:
Try to avoid repeating things that are already clear from the code.  This comment just restates what the code does, so it doesn't add anything.